From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@IlyaGrebnov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tamram commented:
This is a bug that will be fixed soon in the storage emulator. We'd prefer not to publish bug information in this section, so we are declining this PR. Thank you for your submission, however, and your interest in improving our content.

IlyaGrebnov commented:
Is it normal practice to hide unwanted content from ACOM?

PR reflect current state of Azure storage emulator in section dedicated for this type of differences. I also do not buying argument that issue will be fixes soon, because I opened this bug 5 months ago.

jghanem commented:
Hi Ilya,
Thank you for your interest in documenting this bug; we will be looking at a better place to document such unexpected behavior moving forward.
However, as we mentioned to you over an email thread, this bug will be fixed as part of our next  emulator release early this summer.
We apologize for any inconvenience that this might have caused you.

Thanks,
Jean

carolinacmoravia committed:
M10606: Broken link: [Memory optimized]

@v-shiuma, 
Could you review this file, please?
Many thanks!

ruoxinCheng committed:
Add new features to UsqlVScode

cristy committed:
Modifying tutorial to be more succint.

cristy committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sfcreatefix

ruthogunnnaike committed:
Object Transfer Quickstart-Python

ruthogunnnaike committed:
QuickStart Object Transfer - Added Content Body

ruoxinCheng committed:
Update screenshot and content.

ruoxinCheng committed:
Fix the icon

ruoxinCheng committed:
Update passages and fix screenshots.

ruoxinCheng committed:
Fixed Spelling,grammer.

ruoxinCheng committed:
fixed format

ruoxinCheng committed:
Fixed format

ruoxinCheng committed:
Update keywords.

cristy committed:
Made edits based on partner team feedback

cristy committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sfcreatefix

cristy committed:
More edits based on acrolinx

cristy committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sfcreatefix

ruthogunnnaike committed:
Initial Commit

ruthogunnnaike committed:
Initial Commit

ruthogunnnaike committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ObjectTransferPythonBranch

ruthogunnnaike committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ObjectTransferPythonBranch

ruthogunnnaike committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ObjectTransferPythonBranch

ruthogunnnaike committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ObjectTransferPythonBranch

ruthogunnnaike committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ObjectTransferPythonBranch

ruthogunnnaike committed:
Correction

ruthogunnnaike committed:
link update

ruthogunnnaike committed:
Placeholder update

ruthogunnnaike committed:
Corrections

ruthogunnnaike committed:
Correction2

Commit:
task 591: Rewrite add custom domain to CDN endpoint

Commit:
task 591: Fix see also link

Commit:
task 591: remove note and move to beginning of step 2

Commit:
task 591: edit title metadata and update TOC

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-591

Commit:
task 591: rewrite step 5

Commit:
edit pass: azure-stack-version-profiles

Commit:
edit pass: azure-stack-version-profiles

Commit:
task 591: rewrite CNAMES material and add tables

Commit:
task 591: edits

ruthogunnnaike committed:
Python module update

ruthogunnnaike committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ObjectTransferPythonBranch

ruthogunnnaike committed:
Python module update

ruthogunnnaike committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ObjectTransferPythonBranch

ruthogunnnaike committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ObjectTransferPythonBranch

ruthogunnnaike committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ObjectTransferPythonBranch

ruthogunnnaike committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ObjectTransferPythonBranch

ruthogunnnaike committed:
TOC update

ruthogunnnaike committed:
TOC update

ruthogunnnaike committed:
TOC update

anoobbacker committed:
New updates

cristy committed:
More edits based on partner feedback

cristy committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sfcreatefix

anoobbacker committed:
Update storsimple-disaster-recovery-using-azure-site-recovery.md

anoobbacker committed:
ASR image

anoobbacker committed:
Merge branch 'master' of https://github.com/anoobbacker/azure-docs-pr

anoobbacker committed:
Minor corrections

HemantMahawar committed:
Removed error for AzureRM installation from PSGallery

ruoxinCheng committed:
Add status screeshots and update local debug doc

MGoedtel committed:
Updated image for Linux quickstart on LP

dominicbetts committed:
Add OPC publisher article

dominicbetts committed:
Update metadata

mmacy committed:
Update Containers dev center cards

mmacy committed:
Update Containers dev center cards

Commit:
Heidi feedback

Commit:
minor changes

Commit:
Minor fix.

mmacy committed:
[acr] add link to pricing

mmacy committed:
[acr] add link to pricing

bwren committed:
Log Analytics API version

LoriMerrick committed:
Changes made per author's request.

Commit:
fixing links to point to docs.ms instead of github

Commit:
More minor changes.

Commit:
Acrolinx

garyericson committed:
Replacing icon image

sethmanheim committed:
Freshness update

KumudD committed:
added clarification about traffic volume

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TMTRafficViewupdate

sethmanheim committed:
Freshness updates

sethmanheim committed:
Freshness review

Commit:
task 591: Replace temporary step wording with dependent step

Commit:
Updating screenshot.

sethmanheim committed:
Link fixes

PRMerger17 committed:
Merge pull request #25552 from sethmanheim/wk-fresh3

Freshness review

PRMerger19 committed:
Merge pull request #25548 from sethmanheim/wk-fresh2

Freshness update

Commit:
copy edit

sethmanheim committed:
Update per PM feedback

sethmanheim committed:
Forgot to delete old text

rjagiewich committed:
Merge pull request #25506 from mmacy/containers-dev-center-01

Update cards in Containers dev center

PRMerger19 committed:
Merge pull request #25557 from sethmanheim/geodr

Update per PM feedback

ShawnJackson committed:
Update azure-stack-version-profiles.md

bandersmsft committed:
revising CSP quickstart

Commit:
Merge pull request #25522 from bwren/la-apiversion

Log Analytics API version

Commit:
Assigning Azure Code Analysis topic

ShawnJackson committed:
Merge pull request #25231 from LoriMerrick/azure-stack-version-profiles

edit pass: azure-stack-version-profiles

bandersmsft committed:
minor update

PRMerger13 committed:
Merge pull request #25562 from kraigb/master

Assigning Azure Code Analysis topic

Commit:
Merge pull request #25560 from bandersmsft/csp

revising CSP quickstart

ktoliver committed:
Merge pull request #25080 from dksimpson/dean-591

task 591: Rewrite add custom domain to CDN endpoint

ktoliver committed:
Merge pull request #25534 from bradsev/tdsp-cleanup

fixing links to point to docs.ms instead of github

ktoliver committed:
Merge pull request #24351 from cristy/sfcreatefix

Sfcreatefix

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

sanjeev3 committed:
Update facebook-post-moderation.md

KumudD committed:
tweaks

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TMTrafficViewupdate

KumudD committed:
updated traffic monitoring article

PRMerger19 committed:
Merge pull request #25575 from sanjeev3/master

Minor updates to Titles

linda33wj committed:
Fix DB2 LS type for v1

HemantMahawar committed:
Making small style change

Commit:
Update backup-azure-arm-vms-prepare.md

Commit:
Add files via upload

PRMerger15 committed:
Merge pull request #25580 from linda33wj/master

Fix DB2 LS type for v1

JasonWHowell committed:
Update quickstart-create-mysql-server-database-using-azure-portal.md

ruoxinCheng committed:
update vscode doc, local debug doc, and update some screenshots

JasonWHowell committed:
Update quickstart-create-mysql-server-database-using-azure-portal.md

JasonWHowell committed:
Update quickstart-create-mysql-server-database-using-azure-portal.md

PRMerger15 committed:
Merge pull request #25583 from JasonWHowell/patch-1

Update quickstart-create-mysql-server-database-using-azure-portal.md

JasonWHowell committed:
Update quickstart-create-mysql-server-database-using-azure-portal.md

JasonWHowell committed:
Update quickstart-create-mysql-server-database-using-azure-portal.md

Commit:
Update backup-azure-arm-vms-prepare.md

PRMerger12 committed:
Merge pull request #25585 from JasonWHowell/patch-1

Update quickstart-create-mysql-server-database-using-azure-portal.md

dominicbetts committed:
Fix headings

ggailey777 committed:
Replaced Donna with Glenn in metadata

Commit:
Merge pull request #25588 from ggailey777/cleanup

Replaced Donna with Glenn in metadata

mahesh-unnikrishnan committed:
New article on domain join using Resource Manager template

mahesh-unnikrishnan committed:
Add before you begin section.

mahesh-unnikrishnan committed:
acrolinx fixes

anoobbacker committed:
ASR flow image update

ruoxinCheng committed:
Update Jenny's comments

ruoxinCheng committed:
Update wording

Commit:
Fixed typo.

MGoedtel committed:
Updated troubleshooting section

Commit:
Merge pull request #25593 from jimdial/patch-2

Fixed typo.

vhorne committed:
fix table

MGoedtel committed:
Updated statement for error 106

PRMerger19 committed:
Merge pull request #25516 from mmacy/acr-sku-pricing-add

[acr] add link to pricing

PRMerger15 committed:
Merge pull request #25596 from vhorne/integrate-endpoints

fix table

MGoedtel committed:
added sub-section about supported encrypt protocols

mawah committed:
Updates based on LiveFyre feedback

- Removed an unnecessary (and broken) link
- Added example of settings.cfg file after editing

mawah committed:
Update scenario-aerial-image-classification.md

ktoliver committed:
Merge pull request #25547 from garyericson/10-16-replace-image

Replacing icon image

MGoedtel committed:
new image added

carolinacmoravia committed:
Update oracle-design.md

Commit:
Merge pull request #25597 from mawah/patch-2

Updates based on LiveFyre feedback

BryanLa committed:
Add SQL to MSI tutorials

veljko-msft committed:
Update sql-database-metrics-diag-logging.md

MGoedtel committed:
updated link to image

rjagiewich committed:
Merge pull request #24286 from ruoxinCheng/vscode

Add new features to UsqlVScode

veljko-msft committed:
Update sql-database-metrics-diag-logging.md

PRMerger14 committed:
Merge pull request #20244 from carolinacmoravia/patch-362

M10606: Broken link: [Memory optimized]

rjagiewich committed:
Merge pull request #25480 from HemantMahawar/cloud-shell-powershell-2

Removed error for AzureRM installation from PSGallery

Lisaco88 committed:
Merge pull request #25385 from anoobbacker/master

New updates

Lisaco88 committed:
Merge pull request #25498 from MGoedtel/UpdateLP10162017

Updated image for Linux quickstart on LP

carolinacmoravia committed:
CC5901: Fix typo in container-service-draft-up.md #18472

ruthogunnnaike committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ObjectTransferPythonBranch

ruthogunnnaike committed:
Python module update

rjagiewich committed:
Merge pull request #25589 from mahesh-unnikrishnan/master

New article on domain join using Resource Manager template

BryanLa committed:
Update Azure SDK MSI samples

Lisaco88 committed:
Merge pull request #25503 from dominicbetts/suite-publisher

Suite publisher

PRMerger15 committed:
Merge pull request #25603 from veljko-msft/master

Update sql-database-metrics-diag-logging.md

Commit:
Merge pull request #25605 from carolinacmoravia/patch-404

CC5901: Fix typo in container-service-draft-up.md #18472

Lisaco88 committed:
Merge pull request #25581 from trinadhk/patch-34

Update backup-azure-arm-vms-prepare.md

v-shils committed:
Merge pull request #25594 from MGoedtel/UpdateOMSGateway10172017

Updated troubleshooting section

PRMerger17 committed:
Merge pull request #25606 from BryanLa/patch-2

Update Azure SDK MSI samples

Commit:
Merge pull request #25602 from BryanLa/patch-1

Add SQL to MSI tutorials

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-shils committed:
Merge pull request #25507 from roygara/azureSearchBranch

Adding a couple links to unstructured search article.

Lisaco88 committed:
Merge pull request #24398 from ruthogunnnaike/ObjectTransferPythonBranch

Object transfer python branch

v-alje committed:
Merge pull request #25610 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger16 committed:
Merge pull request #25576 from KumudD/TMTrafficViewupdate

Traffic Manager Traffic View updates
